tests/backends/mysql/test_features.py

```diff
@@ -17,3 +18,24 @@ def test_supports_transactions(self):
         with mock.patch('django.db.connection.features._mysql_storage_engine', 'MyISAM'):
             self.assertFalse(connection.features.supports_transactions)
         del connection.features.supports_transactions
+
+    @staticmethod
+    def _reload_modules():
+        reload(base)
+        reload(features)
+
+    def test_skip_locked_no_wait(self):
```

Comment:
I don't think this test is needed. It's enough that we'll run the test on MySQL 8.

Reply:
Does it harm? it is good to keep it. I personally used it to find a bug.